docker-build/SQLBASH.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+
+
+mysql -u root -proot < /home/mu-ting/docker-build/baseball.sql > output.txt
```

Comment:
This method is actually a lot cleaner than what I tried. I ended up making a lot of little mysql connections to run each part of my script,  but a few times it would fail to connect at a step and I would have only part of the calculation done. I think making one connection and running everything at once was a better way to go.